master/docs/manual/configuration/changesources.rst

```diff
@@ -1002,6 +1002,11 @@ The :bb:chsrc:`HgPoller` accepts the following arguments:
     Set encoding will be used to parse author's name and commit message.
     Default encoding is ``'utf-8'``.
 
+``revlink``
+    A function that maps branch and revision to a valid url (e.g. hgweb), stored along with the change.
```

Comment:
you can add some words in this file https://github.com/buildbot/buildbot/blob/master/master/docs/spelling_wordlist.txt

Reply:
Ah, thanks. Will fix.